workspaces/liferay-testray-workspace/client-extensions/liferay-testray-custom-element/src/hooks/useQueryParams.tsx

```diff
@@ -76,14 +76,13 @@ const useQueryParams = () => {
 					? field.resource(parameters)
 					: (field.resource as string);
 
-			const filter = SearchBuilder.eq(
-				'id',
-				serializedFilter[field.name][0]
-			);
+			const filter = SearchBuilder.in('id', serializedFilter[field.name]);
 
 			const resourceFilter = resource.includes('filter=')
 				? resource.replace(/(filter=.*?)(&|$)/, `$1 and ${filter}$2`)
-				: `${resource}&filter=${filter}`;
+				: resource.includes('?')
+				? `${resource}&filter=${filter}`
+				: `${resource}?filter=${filter}`;
```

Comment:
```suggestion
    
				?`${resource}${resource.includes('?') ? '&' : '?'}filter=${filter}`
```
Also, could you check if this makes sense?

Reply:
Thanks for the suggestion :fire: